corona-warn-app/cwa-app-android
# New release info screen logic (EXPOSUREAPP-4591)

This PR adds the logic behind the new release info screen.

How to test:

- Start by installing an older version of the app.
- Install the new app with this branch.
- Upon opening the app the New Release info screen should show up.
- Pressing "Weiter" or the "X" button will take you to the home screen.
- Closing the app and opening it again should not show the New Release screen anymore.
- By going to the menu on the top right of the home screen -> App-Information, a new entry "Neue Funktionen" should be visible at the bottom of the list. Tapping it should open the New Release screen. 
- Pressing "Weiter" or the "X" button will take you back to the App-Information screen.

- Try deleting the app data and do the onboarding again.
- The New Release screen should NOT show up.
